addons/account/models/chart_template.py

```diff
@@ -565,15 +565,23 @@ def _load_template(self, company, code_digits=None, account_ref=None, taxes_ref=
 
     @api.multi
     def create_record_with_xmlid(self, company, template, model, vals):
-        # Create a record for the given model with the given vals and
-        # also create an entry in ir_model_data to have an xmlid for the newly created record
-        # xmlid is the concatenation of company_id and template_xml_id
-        ir_model_data = self.env['ir.model.data']
-        template_xmlid = ir_model_data.search([('model', '=', template._name), ('res_id', '=', template.id)])
-        xml_id = "%s.%s_%s" % (template_xmlid.module, company.id, template_xmlid.name)
-        data = dict(xml_id=xml_id, values=vals, noupdate=True)
-        record = self.env[model]._load_records([data])
-        return record.id
+        return self._create_records_with_xmlid(model, [(template, vals)], company).id
```

Comment:
You know that the returned value of `_load_records` is a recordset of length 1 in this case.
Anyway `.id` returns the id of the *first* record in the recordset.

Reply:
yup, I know that doesn't make much of a difference... just to say that I find the annotation with `[0]`.id more explicit as `.id` of a recordset would throw an error in case of a recordset length > 1

but that's a personal opinion, it's up to you to keep it as it